src/App.js

```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Linters are useful for ensuring we write clean code so please do not disable them.